src/js/ScrollableMessageButtons.js

```diff
@@ -12,9 +13,8 @@ export default class ScrollableMessageButtons extends React.Component {
         if(softButton.systemAction === "STEAL_FOCUS") {
             uiController.onScrollableMessageStealFocus(softButton.msgID, softButton.appID);
         } else if (softButton.systemAction === "KEEP_CONTEXT") {
-            uiController.onScrollableMessageKeepContext(softButton.msgID, softButton.appID, softButton.duration);
-        } else {
```

Comment:
please restore cases as they were, `DEFAULT_ACTION` should not call keepContext

Reply:
@iCollin please find the changes [here](https://github.com/LuxoftSDL/generic_hmi/pull/39/files).